bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -268,7 +268,11 @@ public String getMetadataServiceUriUnchecked() throws UncheckedConfigurationExce
      * @throws ConfigurationException if the metadata service uri is invalid.
      */
     public String getMetadataServiceUri() throws ConfigurationException {
-        String serviceUri = getString(METADATA_SERVICE_URI);
+        List servers = getList(METADATA_SERVICE_URI, null);
```

Comment:
This is not correct, METADATA_SERVICE_URI is a URI, it is not a list of servers

for instance it should start with zk://